Create table

Flunavgt opened the pull request:
On this PR:

- Created database vet_clinic
- Created Table animals
- Insert animal information
- performed multiple Queries.

Pictures of results:
![Screen Shot 2022-10-17 at 15 31 24](https://user-images.githubusercontent.com/99816838/196295066-9bf5b63b-332b-4be9-8c1c-476e6c068e32.png)
![Screen Shot 2022-10-17 at 15 42 54](https://user-images.githubusercontent.com/99816838/196295071-eb0005a2-fb49-48b9-9cea-118b2abe5bd4.png)
![Screen Shot 2022-10-17 at 15 50 12](https://user-images.githubusercontent.com/99816838/196295073-93f4c1c8-e68a-4498-bdbb-266f626a45c6.png)
![Screen Shot 2022-10-17 at 15 58 31](https://user-images.githubusercontent.com/99816838/196295074-cf51ca3e-6728-429a-96eb-28326a383105.png)
![Screen Shot 2022-10-17 at 16 01 25](https://user-images.githubusercontent.com/99816838/196295075-f83704d1-718f-487c-b9f4-2139c5497ce0.png)
![Screen Shot 2022-10-17 at 16 06 18](https://user-images.githubusercontent.com/99816838/196295077-b97f2743-ceca-479e-8b7c-23b29324d484.png)
![Screen Shot 2022-10-17 at 16 13 15](https://user-images.githubusercontent.com/99816838/196295078-4cd93ed9-2ddc-4d25-a8e0-453e2f99261b.png)
![Screen Shot 2022-10-17 at 16 19 02](https://user-images.githubusercontent.com/99816838/196295079-9d8f49c9-a9ab-4f39-8b4a-5f5b7208d09c.png)

Flunavgt committed:
Update schema.sql

Add animals Table

Flunavgt committed:
Update schema.sql

Update table

Flunavgt committed:
Update data.sql

Insert first values into table

Flunavgt committed:
Update queries.sql

Add list of Queries

Flunavgt committed:
Update README.md

Update information

Flunavgt committed:
Create MIT.md

Add MIT

zieeco commented on data.sql:
```diff
@@ -3,3 +3,14 @@
+INSERT INTO animals (
+name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES 
+('Gabumon', '11-15-2018', 2, TRUE, 8), 
```
- Hey, Please, let's make the value for the weight decimal since it's a floating number and not an integer here. `8.0`

zieeco commented on data.sql:
```diff
@@ -3,3 +3,14 @@
+VALUES 
+('Gabumon', '11-15-2018', 2, TRUE, 8), 
+('Pikachu', '01-07-2021', 1, FALSE, 15.04), 
+('Devimon', '05-12-2017', 5, TRUE, 11);
```
- Hey, Please, let's make the value for the weight decimal since it's a floating number and not an integer here. `11.0`

zieeco commented on queries.sql:
```diff
@@ -1,3 +1,12 @@
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
```
- Hey, since we're checking the date_of_birth in a range, we should then extend it to the last day and the last month in the end year as `'2019-31-12';` so that the analysis will fall between this range as opposed to you just specifying the first day and first month here `'2019-01-01'`

zieeco reviewed (changes requested):
Hi @Flunavgt,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice setup. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


<img src="https://media0.giphy.com/media/8UGVQESpDMMQl7yd3F/giphy.gif"/>


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Flunavgt commented:
Updated Weight on suggested animals:
![Screen Shot 2022-10-17 at 17 16 01](https://user-images.githubusercontent.com/99816838/196300860-3ac9cc24-cb43-4334-ac09-97851c210083.png)

Updated date range on the query:
![Screen Shot 2022-10-17 at 17 19 49](https://user-images.githubusercontent.com/99816838/196301232-4eb39537-0834-4e21-981b-c626b3dc5895.png)

Flunavgt committed:
Update queries.sql

Update on date range on query

codecaiine reviewed (approved):
Hi @Flunavgt , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Nice setup. :heavy_check_mark:
Descriptive PR. :heavy_check_mark:
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
